app/Models/BeatmapSet.php

```diff
@@ -684,4 +686,18 @@ public function approver()
     {
         return $this->belongsTo("App\Models\User", 'user_id', 'approvedby_id');
     }
+
+    public function description()
+    {
+        $topic = Topic::find($this->thread_id);
+        $post = Post::find($topic->topic_first_post_id);
+
+        $split = preg_split('[-{15}]', $post->post_text);
```

Comment:
just noticed, hopefully there's no description which contains another `---------------` otherwise anything after that will be gone (since they'll be part of `$split[2]` and up)...